dolphin/processor/config.py

```diff
@@ -81,7 +81,23 @@ def pixel_size(self):
         :return:
         :rtype:
         """
-        return self.settings['pixel_size']
+        if type(self.settings['pixel_size']) == float:
+            return [self.settings['pixel_size']]
+        else:
+            return self.settings['pixel_size']
+
+    @property
+    def maximum_pixel_size(self):
```

Comment:
This function should return just a float, not a list, I think. Because, maximum value is always just one number.